Makefile

```diff
@@ -13,8 +13,9 @@ install:
 
 
 .PHONY: run
-run: migrate ## Run the app
+run: ## Run the app
 	. ./dev_env.sh && \
+	poetry run python -c 'from hushline import create_app; from hushline.db import db; from sqlalchemy import text; create_app().app_context().push(); db.session.execute(text("CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\"")); db.session.commit(); db.create_all()' && \
```

Comment:
We probably need two make targets tbh because doing migrations as part of the normal dev workflow is horrendous for rapid testing IMO. I can set up two scripts for this.